### ISSUE TITLE
a-output-event name and type as the same variable

### ISSUE THREAD

**Issue by Gelio** [johnpapa/vscode-angular-snippets#56]
`a-output-event` snippet currently is as follows:
``` typescript
@Output() eventName: EventEmitter<eventType> = new EventEmitter<eventType>();
```
There is only one variable for this snippet that changes both the `eventName` and the `eventType`, which I believe is not a desired result, since you end up with an output named `boolean` or an `EventEmitter` of type `confirm`.

I may create a pull request to change this if you would like, but first I wanted to point it out.

Great job with those snippets, by the way. They speed up the process a lot!

**Comment by johnpapa** [comment]
good catch, yes, please make a PR!

**Comment by Gelio** [comment]
I have created a PR for this issue (#57). Thank you for responding, It's an honor to be able to talk to you, not to mention contribute to one of your projects! I'm a big fan, basically.

I was also wondering if the type annotation for this `EventEmitter` isn't redundant. We already have type inference since we're assigning an instance of `EventEmitter` to this property and when it comes to other properties with an explicit type annotation _tslint_ shows an error, for instance:
![tslint-error](https://i.gyazo.com/52650b7a94306c96b4a32cf5966f5099.png)

It may not be the same since `EventEmitter` is not a primitive type, but I reckon that dropping that annotation would result in a bit tidier codebase. What do you think?

**Comment by johnpapa** [comment]
i removed the extra type too. thanks

published in 2.3.3
